### PR TITLE
[tests-only] [full-ci] Run phan with PHP 7.3

### PR DESCRIPTION
**Pull request by phil-davis**
`phan` will report errors if any of the code is not compatible with PHP 7.3.

This app still supports PHP 7.3 when used with oC10 core 10.11 and earlier.

To get `phan` passing, I suppressed most of the things it reported. They all look like OK code to me. There are a few deprecated constants, but I don't want to change them to their newer names, because that might break if running with some older core version with older dependenices or...

### CODE REVIEW

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/files_antivirus/2802/5/3
`phan` passed with PHP 7.3 (and 7.4) - good

File: .phan/config.php [hunk 1]
```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * @author Patrick Jahns <github@patrickjahns.de>
+ *
+ * @copyright Copyright (c) 2018, ownCloud GmbH
```
Comment (kiranparajuli589) [inline .phan/config.php:5]:
```suggestion
 * @copyright Copyright (c) 2022, ownCloud GmbH
```

File: .phan/config.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * @author Patrick Jahns <github@patrickjahns.de>
```
Comment (kiranparajuli589) [inline .phan/config.php:3]:
```suggestion
 * @author Patrick Jahns <github@patrickjahns.de>
```
:arrow_up:

Comment (phil-davis) [reply .phan/config.php:3]: This is a copy-paste of the config from other repos, originally written by Patrick. I don't know what the rules are when doing that sort of thing. In the other repos, I have been leaving his name.

**Review by kiranparajuli589 — APPROVED** [review]

**Pull request merged**